stellarphot/tests/test_core.py

```diff
@@ -1369,3 +1370,14 @@ def test_to_lightcurve_multiple_passbands(simple_photometry_data):
         match=r"Passband SI not found for this star",
     ):
         two_filters.lightcurve_for(1, passband="SI")
+
+
+def test_reading_2_0_0_alpha_photometry_file():
+    """
+    Just make sure that old photometry files are still readable.
```

Comment:
Is this the old or the new format when you refer to 2.0.0alpha, I am guessing old, but if it is unchanged from the last version 1 settings, I think it makes more sense to call this `test_reading_v1_photometry_file`.

Reply:
"old" here refers to `2.0.0alpoha`, which I've been running on jupyter3 since last August. That has been an adventure....I wanted to make sure that updating to a 2.0 release won't stop research students in their tracks.